app/src/main/java/com/triviagenai/triviagen/trivia/presentation/results/components/TriviaScoreIndicator.kt

```diff
@@ -0,0 +1,40 @@
+package com.triviagenai.triviagen.trivia.presentation.results.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.res.dimensionResource
+import com.triviagenai.triviagen.R
+import com.triviagenai.triviagen.trivia.presentation.TriviaQuestionViewModel
+import com.triviagenai.triviagen.ui.theme.TriviaGreen
+import com.triviagenai.triviagen.ui.theme.TriviaRed
+
+@Composable
+fun TriviaScoreIndicator(
+    score: Float,
+    questionsSize: Int,
+) {
+    Box(
+        contentAlignment = Alignment.Center
+    ) {
+        CircularProgressIndicator(
+            progress = score / (TriviaQuestionViewModel.POINTS * questionsSize),
+            color = if (score > 0.5f) TriviaGreen else TriviaRed,
```

Comment:
I'm not sure I saw the colour as red when the score was below 0.5f
Score may need to be a remembered value. You could try something like:
`val score: Float by remember { mutableIntStateOf(0f) }`